### PR TITLE
Add peribolos job for updating knative-sandbox

### PR DESCRIPTION
**Pull request by chizhg**
**What this PR does, why we need it**:
In preparation for https://github.com/knative/community/pull/135. The jobs here should be added after that PR is merged.
@evankanderson please let me know if you want to enable more `fix`es.

/cc @evankanderson @coryrc @chaodaiG

### CODE REVIEW

File: config/prod/prow/jobs/custom/peribolos.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Copyright 2020 The Knative Authors
```
Comment (chaodaiG) [inline config/prod/prow/jobs/custom/peribolos.yaml:1]: would make more sense to me renaming this file to `knative-sandbox.yaml`

Comment (chizhg) [reply config/prod/prow/jobs/custom/peribolos.yaml:1]: The jobs here are running for the `knative/community` repo, so I feel `knative-sandbox.yaml` might not be a good name. How about `peribolos-knative-sandbox.yaml`?

Comment (chaodaiG) [reply config/prod/prow/jobs/custom/peribolos.yaml:1]: Doesn't really matter that much

/lgtm

**Comment by chizhg** [comment]
/unhold
Now that knative/community#135 has been merged.

**Pull request merged**